mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -11843,3 +11843,103 @@ resource "google_container_cluster" "with_autopilot_gcp_filestore" {
 }
 `, name)
 }
+
+func TestAccContainerCluster_withDnsEndpoint(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
```

Comment:
Could I get you to break these into functions, similar to other tests in the file / the structure in https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test? Apologies, I see why this makes sense in the context of a single change, but it's a big codebase so we want to stay consistent.

Reply:
Sure, consistency is a good thing.